pywemo/ouimeaux_device/dimmer.py

```diff
@@ -42,8 +42,11 @@ def set_brightness(self, brightness):
         self._brightness = int(brightness)
 
     def subscription_update(self, _type, _param):
-        """Disable subscription updates."""
-        return False
+        """Update the dimmer attributes due to a subscription update event."""
+        if _type == "Brightness" and self._state:
```

Comment:
My only question is whether this should be `Brightness` or `brightness`.  From the dimmer YAML file, I see `<relatedStateVariable>brightness</relatedStateVariable>`

Reply:
ah, never, this is going to come from HA which uses capital `Brightness` it looks like.